magento/pwa-studio
# Update devdoc dependencies

## Description

Update devdoc dependencies

## Related Issue





Closes #ISSUE_NUMBER.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps

1. Navigate to the devdocs project
2. Run an audit: `yarn audit`
3. Verify 0 packages needed to update

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
